orc/src/main/java/org/apache/iceberg/orc/OrcValueReaders.java

```diff
@@ -233,4 +236,16 @@ public void setBatchContext(long newBatchOffsetInFile) {
       this.batchOffsetInFile = newBatchOffsetInFile;
     }
   }
+
+  private static class DeleteMarkReader implements OrcValueReader<Boolean> {
```

Comment:
Is there a constant reader that could be reused?

Reply:
We could use the existing constant reader from parquet and orc readers. I also created a new constant reader for Avro reader.